src/content/docs/core-concepts/multi-proofs.md

```diff
@@ -4,3 +4,9 @@ description: Core concept page for "Multi-proofs".
 ---
 
 A great resource to learn about Taiko's approach to security with multi-proofs is the Twitter thread [here](https://x.com/taikoxyz/status/1745546868028068273).
+
+Cryptographic implementations are complex and not yet mature. To minimize potential bugs and vulnerabilities, diversity in proof systems is needed. Taiko is one of the advocates who strongly defends the multi-proofs approach in rollups. Taiko's approach aims to increase security and diversity by using multiple proof systems and clients, thus reducing the risk associated with bugs or vulnerabilities in any single system. The approach also includes the integration of different types of proofs.
+
+Taiko proposes a robust multi-proof pipeline by translating assembly-level instructions coming from different execution clients into arithmetizations for algebraic or polynomial proof systems. Different backends to encode these arithmetizations, such as SuperNova, Halo2, and eSTARK can also be used, without being limited by using a single protocol.
```

Comment:
```suggestion
Taiko proposes a robust multi-proof pipeline with [powdr labs](https://www.powdr.org/), which translats assembly-level instructions coming from different execution clients into arithmetizations for algebraic or polynomial proof systems. Different backends to encode these arithmetizations, such as SuperNova, Halo2, and eSTARK can also be used, without being limited by using a single protocol.
```
Gotta mention powdr here cuz the modular ZkVM is their project, we just use it in our proving stack 😅